### DISCUSSION TITLE
How can I switch off this annoying orange code hint for validation?

### DISCUSSION THREAD

**Discussion by fishmi** [yandeu/five-server#68] [Help / Questions]
How can I switch off this annoying orange code hints for validation?
It must be possible to use this extension with files that do not contain standard elements.
Thanks for the help.

Martin
![Snipaste_2022-10-19_09-02-39](https://user-images.githubusercontent.com/105578/196620658-24d92d5b-7522-4202-8b2c-dcd632645e65.png)

**Comment by yandeu** [comment]
At the moment you can't.

Feel free to tweak the config in:
https://github.com/yandeu/five-server/blob/d39fac4d2d4ad53873b9c6327e64fd536cf69b7a/src/workers/parseBody.ts#L18

---

If you don't use `injectBody` in your config file, five server will not validate your html.

---

Btw, if you use `<my-include />`, which is a valid html element, it should work.

**Comment by fishmi** [comment]
Thank you very much.

I have now commented out the line
"const report = htmlvalidate.validateString(html)"
in the file "parseBody.js".

Then the hints are no longer displayed.

injectBody is needed and I can theoretically rename the tag, but I would like to avoid that because the consequences would be too extensive.

It would be great if this could be configured in the future.

**Reply by yandeu** [reply]
Maybe it is easier to just disable this rule: https://html-validate.org/rules/element-name.html
